sdk/hybriddatamanager/Microsoft.Azure.Management.HybridDataManager/src/Generated/HybridDataManagementClient.cs

```diff
@@ -361,7 +357,7 @@ private void Initialize()
             DataStoreTypes = new DataStoreTypesOperations(this);
             PublicKeys = new PublicKeysOperations(this);
             BaseUri = new System.Uri("https://management.azure.com");
-            ApiVersion = "2016-06-01";
+            ApiVersion = "2019-06-01";
```

Comment:
As your are changing the API version, please also update the api tags in this file: `AzSdk.RP.props`

Reply:
By the way, does this API version update bring any breaking changes? If so, SDK will need a major version update.

Reply:
The AzSdk.RP.props file is also changed in this PR

Reply:
There is no breaking change in this API version upgrade. We are adding a new value to an enum and have updated some comments about the data models/APIs in use.